src/linked-list.js

```diff
@@ -4,6 +4,41 @@ class LinkedList {
     this.tail = null;
     // Do not modify anything inside of the constructor
   }
+  addToTail(newValue) {
+    const node = {
+      value: newValue,
+      next: null
+    };
+    if (this.head === null) {
```

Comment:
Start getting in the habit of using the `!` operator when you want to check if something is undefined or null. It is more concise and more flexible than checking if something is strictly just undefined.